Resources/Prototypes/Entities/Objects/Misc/dat_fukken_disk.yml

```diff
@@ -5,6 +5,7 @@
   description: A nuclear auth disk, capable of arming a nuke if used along with a code. Note from nanotrasen reads "THIS IS YOUR MOST IMPORTANT POSESSION, SECURE DAT FUKKEN DISK!"
   components:
   - type: NukeDisk
+  - type: Geofencing
```

Comment:
`Geofenced` or `ProtectedByGeofencing` or whatever. `Geofencing` implies this component geofences something else.